src/CoiniumServ/Transactions/GenerationTransaction.cs

```diff
@@ -158,7 +158,7 @@ public GenerationTransaction(IExtraNonce extraNonce, IDaemonClient daemonClient,
                             blockTemplate.CoinBaseAux.Flags,
                             TimeHelpers.NowInUnixTimestamp(),
                             (byte) extraNonce.ExtraNoncePlaceholder.Length,
-                            "/CoiniumServ/")
+                            "/Mined by bit2pool.com/") //Changed a mined message from CoiniumServ to mining pool name.
```

Comment:
don't change this please on the main repo.

Reply:
@bonesoul, Yes, I will return it back..